### DISCUSSION TITLE
Docker image support for linux/arm/v7

### DISCUSSION THREAD

**Discussion by ricardoboss** [datalust/seq-tickets#1759] [Help] [answered]
Hi there 👋 

Would it be possible to provide support for linux/arm/v7 platforms in Docker?
I'd like to run Seq on my Raspberry Pi.

**Comment by KodrAus** [comment, marked as answer]
Hi @ricardoboss! If I understand correctly, the armv7 architecture is 32bit, but Raspberry Pis can also run 64bit armv8. If you're running Raspbian I think you get 32bit by default, but you can flash a 64bit armv8 image too.

**Reply by KodrAus** [reply]
For what it's worth, our very first ARM builds of Seq were built on a Raspberry Pi 4b I kept on my desk 🙂

**Reply by ricardoboss** [reply]
Ah I see. Yes, I am running Raspbian on a Pi 4b, but it is 32 bit. I'll see how much effort it would be to flash a new image and restore all my services then... Thanks!
